cmd/syncEnumerator.go

```diff
@@ -55,12 +55,14 @@ func (cca *cookedSyncCmdArgs) initEnumerator(ctx context.Context) (enumerator *s
 		}
 	}
 
+	includeDirStubs := cca.fromTo.From().SupportsHnsACLs() && cca.fromTo.To().SupportsHnsACLs() && cca.preservePermissions.IsTruthy()
```

Comment:
Would it make sense to replace the isHNSToHNS option cookedSyncCmdArgs to have includeDirStubs?

Reply:
isHNStoHNS hasn't existed for an extended period of time.